rollup.config.js

```diff
@@ -39,18 +32,20 @@ const configPlugin = {
 
 module.exports = [
   {
-    input: "src/lib/app.js",
+    input: "src/lib/bootstrap.js",
     output: {
       dir: "dist",
-      format: "iife",
-      entryFileNames: outputPattern,
-      chunkFileNames: outputPattern,
+      format: "esm",
+      chunkFileNames: "_[hash].js",
```

Comment:
It could be. In that case it generates two types of files: "measure.[hash].js" as well as lots of "chunk.[hash].js". It doesn't really matter in practice—are we going to care what our dist JS bundles are named?

Reply:
I found it helpful just now because I was curious what went into each bundle.